### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by ankitskvmdam**
1) In README.md
	In ia-prototype-apps section instead of ia-prototypes-apps, prototypes is written.

2) In packages/ia-prototype-apps/README.md
	Instead of ia-prototype-apps, prototypes is written.

In both cases, the name of the folder is wrong in README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,13 +61,13 @@ See [packages/ia-prototype-apps/README.md](packages/ia-prototype-apps/README.md)
 
 Prototypes can be run like this:
 ```
-cd packages/prototypes
-yarn run parcel prototypes/examples/example-hello/index.html
+cd packages/ia-prototypes-apps
```
Comment (iisa) [inline README.md]: we will leave directory name the same, let's update these to reflect directory name.

Comment (ankitskvmdam) [reply README.md]: I didn't understand.

**Comment by akshaykumar23399** [comment]
Can you explain what you really wanted did you wanted to clear all  or to change it

File: README.md (hunk 1 above)
Comment (iisa) [reply README.md]: the directory is `packages/ia-prototype-apps`. please reflect this in your code.

**Comment by iisa** [comment]
hi @ankitskvmdam @akshaykumar23399 is anyone working on this?  The repo reference should be the same as the directory.

File: README.md (hunk 1 above)
Comment (ankitskvmdam) [reply README.md]: I haven't made any change in the directory structure. I only made changes in the README.md file.

**Comment by iisa** [comment]
@ankitskvmdam @akshaykumar23399 going to close for now. Feel free to restart and open when you have availability.

**Pull request closed without merging**